helm/operations-engineering-example/templates/ingress.yml

```diff
@@ -3,15 +3,15 @@ kind: Ingress
 metadata:
   name: example-ingress
   annotations:
-    external-dns.alpha.kubernetes.io/set-identifier: example-ingress-${NAMESPACE}-green
+    external-dns.alpha.kubernetes.io/set-identifier: {{ .Values.ingress.identifier | quote }}
     external-dns.alpha.kubernetes.io/aws-weight: "100"
 spec:
   ingressClassName: default
   tls:
     - hosts:
-        - ${NAMESPACE}.apps.live.cloud-platform.service.justice.gov.uk
+        - {{ .Values.ingress.host }}
```

Comment:
love the parameterisation here 💖 makes the template a lot more re-useable, and also easier to read at the configuration level 😁

Reply:
yes will param further